src/components/PastTripForm.js

```diff
@@ -0,0 +1,56 @@
+import { useState } from 'react';
+import styled from 'styled-components';
+import ButtonAdd from './Button';
+
+export default function PastTripNotes({ onHandleNewNote }) {
+  const [textInput, setTextInput] = useState('');
+
+  return (
+    <>
+      <NotesForm
+        aria-labelledby="notes-form"
+        onSubmit={handleSubmit}
+        autoComplete="off"
+      >
+        <Label htmlFor="notes">Things to remember:</Label>
+        <Input
+          type="text"
+          id="notes"
+          name="notes"
+          placeholder="Enter your notes..."
+          value={textInput}
+          onChange={event => setTextInput(event.target.value)}
+          maxLength="500"
+          rows="3"
+          required
+        />
+        <ButtonAdd variant="add" type="submit" id="notes-form">
+          Add notes
+        </ButtonAdd>
+      </NotesForm>
+    </>
+  );
+
+  function handleSubmit(event) {
+    event.preventDefault();
+    const form = event.target;
+    const inputElement = form.elements.notes.value.trim();
```

Comment:
"inputElement" sounds like the entire HTML element, better use something like "inputValue" here?